src/objective-c/GRPCClient/private/GRPCSecureChannel.m

```diff
@@ -33,12 +33,24 @@
 
 #import "GRPCSecureChannel.h"
 
-#import <grpc/grpc_security.h>
+grpc_credentials *CertificatesAtPath(NSString *path) {
```

Comment:
That was my first thought, but then to load the default certs, which I wanted to cache (to not read the file every time), we need to call this function. And making it call itself recursively and cache the result was very convoluted, so I put the default + caching logic outside.

The function needs to be static, though; I didn't mean to export it out of this file.